services/net/transcription/TNO.Services.Transcription.csproj

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\libs\net\services\TNO.Services.csproj" />
     <ProjectReference Include="..\..\..\libs\net\kafka\TNO.Kafka.csproj" />
+    <ProjectReference Include="..\..\..\libs\net\dal\TNO.DAL.csproj" />
```

Comment:
true, this is because the s3 helper class is in DAL. Transcript service is not really using database directly.
I will move the s3 class to another project.
thanks